src/data/nav.yml

```diff
@@ -146,6 +146,22 @@
           url: '/build-apps/ab-test/subscribe'
     - title: See all open source apps
       url: https://opensource.newrelic.com/nerdpacks/
+
+- title: Instant observability
+  icon: nr-build-apps
+  url: '/instant-observability'
+  pages:
+    - title: Build a quickstart
+      url: '/instant-observability/build-a-quickstart'
+      pages: 
+        - title: Send data to New Relic
```

Comment:
The icon should be changed to not have any confusion by using the same icon as the `build apps` navigation item

Reply:
Sorry, I advised Mahreen that this didn't really matter much since we're probably going to have to re-organize anyway with all the new lab topics coming. Not to mention the site migration work. @mehreentahir16 Please ignore what I said earlier.